Library/Homebrew/test/caveats_spec.rb

```diff
@@ -26,4 +27,90 @@ def caveats
       expect(described_class.new(f)).not_to be_empty
     end
   end
+
+  describe "#python_caveats" do
+    subject { described_class.new(f).send(:python_caveats) }
+
+    let(:packages27) { f.lib/"python2.7/site-packages" }
+    let(:packages34) { f.lib/"python3.4/site-packages" }
+
+    before do
+      f.prefix.mkpath
+      f.opt_prefix.parent.mkpath
+      FileUtils.ln_s f.prefix, f.opt_prefix
+    end
+
+    context "python sees our site-packages" do
+      before do
+        allow(Language::Python).to receive(:reads_brewed_pth_files?).and_return(true)
+      end
+      context "site-packages is empty" do
+        it("should return nil") { is_expected.to be_nil }
```

Comment:
I'm brand new to Rspec-ing; does this mean you'd rather see `it { is _expected.to be_nil }`?